astro/src/content/docs/get-started/start-here/index.mdx

```diff
@@ -0,0 +1,205 @@
+<DockerInstallAndStart codeRoot={frontmatter.codeRoot} kickstartUri={frontmatter.codeRoot + "/kickstart/kickstart.json"}/>
+</details>
+
+### Install and Run in GitPod
+<details>
+<summary>GitPod</summary>
+<GitPodStart codeRoot={frontmatter.codeRoot} kickstartUri={frontmatter.codeRoot + "/kickstart/kickstart.json"}/>
+</details>
+
+### What Did You Just Do?
+
+You launched a FusionAuth database and server either using Docker or in GitPod, using a kickstart to pre-populate the application details in the FusionAuth database.
+
+## Delegate Authentication
+
+Once you have FusionAuth up and running, you need to set up your application or applications to delegate to FusionAuth. For this guide, there's an example `Start Here` application which is already set up to do so. Let's look at that configuration and then start it up so you can log in.
+
+### Open the Admin Interface
+
+* In the previous step, you logged in to the system as an admin user.  Go back to that browser screen, which should be showing the administrative interface for FusionAuth.
+* Navigate to <InlineUIElement>Applications</InlineUIElement> in the left-hand sidebar. You may need to use the hamburger icon at the top left of the screen to see all options.
+* Choose the `Start Here` application.
+* Choose <InlineUIElement>Edit</InlineUIElement> from the <InlineUIElement>Actions</InlineUIElement> dropdown.
+
+### Application Configuration
+
+After the user signs in at FusionAuth, they need to get back to your application. In the FusionAuth application configuration you can view authorized redirect URLs and how tokens are generated/exchanged.
+
+![The edit screen of the application.](/img/docs/get-started/start-here/edit-application.png)
+
+### After the User Authenticates
+
+To understand how the JavaScript application integrates with FusionAuth, take a quick look at some of the code from the repository.
+
+We craft a URL that includes the `clientId` from the configuration you reviewed above, as well as some other parameters. This is where the user goes to log in with FusionAuth. TODO need to only show the `redirect(302, `fusionAuthURL/oauth2/authorize` line
+
+<RemoteCode url="https://raw.githubusercontent.com/synedra/fusionauth-example-express-start-here/main/app/src/index.ts" lang="javascript" tags="login" />
+
+Once the user authenticates, the code at the configured redirect URL is called. This code sends parameters from the login event (the authorization code), the client identifier (clientId) and other information to get a valid token from FusionAuth. When the app receives this token, the user has fully been authenticated. TODO need to only show the try block here
+
+<RemoteCode url="https://raw.githubusercontent.com/synedra/fusionauth-example-express-start-here/main/app/src/index.ts" lang="javascript" tags="oauth-redirect" />
+
+Once you have the token, what should the app do with it? One option is to store it as a secure cookie. We do that here. TODO show cookie saving
```

Comment:
Need to update the code excerpt (the tags in the repo files) and remove the TODO here.

Reply:
What does it mean to show cookie saving?  Is there specific code you want to show?  There are several different pieces of code surrounding the cookie stuff, or do you want to write something separate?

Reply:
I took care of this. hopefully it is decently clear in the revised doc.